### PR TITLE
Show Plan Upgrade information

### PR DESCRIPTION
**Pull request by fjvallarino**
Ready for final code review. I'm currently trying to stage one last time on stage-1 for validation (let me know if you also want to validate there, otherwise I will merge after PR approval).

@Rise-Vision/apps please review

### CODE REVIEW

**Comment by ezequielc** [comment]
@fjvallarino I am taking a final look. Thanks

**Comment by fjvallarino** [comment]
@ezequielc sure. It's already on stage-1 (the deployment is done even if the tests fail). I'm investigating the e2e failures, but it can be reviewed.

**Comment by ezequielc** [comment]
@fjvallarino Functionality looks good. Thanks!

File: gulpfile.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ var unitTestFiles = [
   "web/bower_components/common-header/dist/js/components/timeline.js",
   "web/bower_components/common-header/dist/js/components/message-box.js",
   "web/bower_components/common-header/dist/js/components/stop-event.js",
+  "web/bower_components/common-header/dist/js/components/plans.js",
```
Comment (alex-deaconu) [inline gulpfile.js]: @fjvallarino I think this is part of the common-header.js code, no need to add again. Thanks.

Comment (fjvallarino) [reply gulpfile.js]: @alex-deaconu should we try removing the other components' lines? They should also be part of common-header.js.

File: web/scripts/app.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ angular.module('risevision.apps', [
     'risevision.common.components.distribution-selector',
     'risevision.common.components.background-image-setting',
     'risevision.common.components.message-box',
+    'risevision.common.components.plans',
```
Comment (alex-deaconu) [inline web/scripts/app.js]: @fjvallarino same comment, I think this is being imported by CH. Thanks.

File: gulpfile.js (hunk 1 above)
Comment (alex-deaconu) [reply gulpfile.js]: @fjvallarino I wouldn't risk it for this release. Thanks.

**Pull request merged**